i18n.js

```diff
@@ -1081,8 +1087,9 @@ module.exports = (function() {
       localeFile = fs.readFileSync(file);
       try {
         // parsing filecontents to locales[locale]
-        locales[locale] = JSON.parse(localeFile);
+        locales[locale] = parser(localeFile);
       } catch (parseError) {
+        console.log(parseError)
```

Comment:
please just remove that console.log - logError should be sufficiant

plus the missing ';' keeps ci failing

Reply:
My bad, debugging leftover...